docs/cloud/satellites.md

```diff
@@ -67,11 +67,14 @@ Follow the steps in the [Earthly Cloud overview](./overview.md#getting-started)
 
 ### 2. Purchase a Satellites Plan
 
-Satellites are now a paid feature and require a valid subscription to begin using. The subscription includes a 14-day free trial which can be canceled before any payment is made. Visit the [pricing page](https://earthly.dev/pricing) for more billing details.
+Satellites are a paid feature which require a subscription to begin using. The subscription includes a 14-day free trial which can be canceled before any payment is made. Visit the [pricing page](https://earthly.dev/pricing) for more billing details.
+
+You can start your free trial by using the checkout form below. Be sure to provide the name of your Earthly org from Step 1.
```

Comment:
I see that this now includes a new field for the org. I should probably update the automated email in Zapier so that it no longer asks for this.

Reply:
Yes, I was thinking that too.